doc-content/enterprise-only/optimizer/proc-quarkus-packaging.adoc

```diff
@@ -0,0 +1,32 @@
+[id='proc-quarkus-packaging_{context}']
+
+
+= Packaging and running the {ProductName} Getting Started application
+After you compile your {ProductName} Getting Started project, you can package it in a JAR file and run it from the command line.
+
+.Prerequisites
+* You have compiled the {ProductName} Getting Started project.
+
+.Procedure
+. To package your {ProductName} Getting Started project, enter the following command in the `root` directory:
++
+[source]
+----
+./mvnw package
+----
++
+This command produces the following JAR files in the `/target` directory:
++
+* `getting-started-1.0-SNAPSHOT.jar`: Contains the classes and resources of the projects. This is the regular
+artifact produced by the Maven build.
+* `getting-started-1.0-SNAPSHOT-runner.jar`: Is an executable JAR file. Be aware that this file is not an uber-JAR file because the dependencies are copied into the `target/lib` directory.
+
+. If development mode is running, press *CTRL+C* to stop development mode. If you do not do this, you will have a port conflict.
```

Comment:
[translatability] Rephrase "If you do not do this":

s/If you do not do this/If development mode is running, you will have a port conflict.

Perhaps "If you don't do this"?